src/binaryen-c.cpp

```diff
@@ -2063,7 +2063,7 @@ void BinaryenSetDebugInfo(int on) {
     std::cout << "  BinaryenSetDebugInfo(" << on << ");\n";
   }
 
-  globalPassOptions.debugInfo = bool(on);
+  globalPassOptions.debugInfo = on ? true : false;
```

Comment:
was this necessary?

Reply:
MSVC emitted a warning there, converting an int to a bool. Not sure why.

Reply:
drive by comment: `on != 0` or `!!on` are the ways I usually see this written.

Reply:
Thanks, `on != 0` is what I used now.